lib/split-home-site-urls/login.php

```diff
@@ -12,11 +12,11 @@ function rewrite_admin_url_to_site_url( $redirect_to ) {
 	}
 
 	// Grab the path relative to wp-admin
-	preg_match( '#/wp-admin(/)?(.+?)$#i', $redirect_to, $admin_paths );
+	$exploded = explode( '/wp-admin', $redirect_to );
```

Comment:
On `explode()` here, we should set a limit of `1`, on the off chance that other parts of the url contain it, like in a GET param.